### PR TITLE
fix(docker): deploy user actions, even if no system actions were deployed

### PR DESCRIPTION
**Pull request by Masterchen09**

### CODE REVIEW

File: docker/datahub-actions/start.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ else
 fi
 
 # Deploy User Actions
-if [ "$(ls -A /etc/datahub/actions/system/conf/)" ]; then
+if [ "$(ls -A /etc/datahub/actions/conf/)" ]; then
```
Comment (jjoyce0510) [inline docker/datahub-actions/start.sh:41]: What is the purpose of this change?

Comment (jjoyce0510) [reply docker/datahub-actions/start.sh:41]: User actions should be deployed regardless of system actions, already. Is this not the case?

Comment (Masterchen09) [reply docker/datahub-actions/start.sh:41]: According to the script there are two kinds of actions or configurations for actions: User and System Actions

These actions or the configurations for these actions are located in the folders /etc/datahub/actions/system/conf and /etc/datahub/actions/conf. The first block in the script handles the System Actions and the second block the User Actions.

Unfortunately the if for the second block is checking the content of the folder for the System Actions. Therefore configuration files for User Actions may not be evaluated if there are no configuration files for System Actions. The if for the second block must check the content of the /etc/datahub/actions/conf folder and not the /etc/datahub/actions/system/conf folder.

Of course this does not impact the "default" configuration of the docker image but may have some influence if you use the datahub-actions docker image as a base image.

**Review by jjoyce0510 — APPROVED** [review]
